cmd/cli_changelog.go

```diff
@@ -0,0 +1,28 @@
+package cmd
+
+import (
+	"github.com/urfave/cli"
+
+	"github.com/Scalingo/cli/cmd/autocomplete"
+	"github.com/Scalingo/cli/update"
+)
+
+var (
+	cliChangelogCommand = cli.Command{
+		Name:     "cli-changelog",
```

Comment:
nitpick: `cli-changelog` seems a bit clumsy. What do you think about just naming it `changelog`?